packages/@react-spectrum/dropzone/docs/DropZone.mdx

```diff
@@ -0,0 +1,374 @@
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/dropzone';
+import {HeaderInfo, PropTable, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/dropzone/package.json';
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import DropZoneAnatomy from './images/DropZoneAnatomy.svg';
+import styles from '@react-spectrum/docs/src/docs.css';
+
+```jsx import
+import {DropZone} from '@react-spectrum/dropzone';
+import {Heading} from '@react-spectrum/text';
+import {Content} from '@react-spectrum/view';
+import {IllustratedMessage} from '@react-spectrum/illustratedmessage';
+import {Text} from 'react-aria-components';
+import {FileTrigger} from 'react-aria-components';
+import {Button} from '@react-spectrum/button';
+```
+
+---
+category: Drag and drop
+---
+
+# DropZone
+
+<PageDescription>{docs.exports.DropZone.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['DropZone']}
+/>
+
+## Example
+
+```tsx example
+import Upload from '@spectrum-icons/illustrations/Upload';
+
+<DropZone
+  maxWidth="size-3000">
+  <IllustratedMessage>
+    <Upload />
+    <Heading>
+      <Text slot="label">
+        Drag and drop your file
+      </Text>
+    </Heading>
+  </IllustratedMessage>
+</DropZone>
+```
+
+
+## Content
+
+A standard DropZone has the following anatomy:
+
+<div className={styles.exampleImage}>
+  <DropZoneAnatomy />
+</div>
+
+A drop zone accepts an [IllustratedMessage](IllustratedMessage.html) as a child which is comprised of three areas:  an illustration, a title, and a body. Each of these sections can be populated by providing the following components to the IllustratedMessage as children: a SVG, a [Heading](Heading.html) (title), and a [Content](Content.html) (body). A [FileTrigger](../react-aria/FileTrigger.html) is commonly paired with a DropZone to allow a user to choose files from their device. 
+
+```tsx example
+<DropZone
+  maxWidth="size-3000">
+  <IllustratedMessage>
+    <Upload />
+    <Heading>
+      <Text slot="label">
+        Drag and drop here
+      </Text>
+    </Heading>
+    <Content>
+      <FileTrigger>
+        <Button variant="primary">Browse</Button>
+      </FileTrigger>
+    </Content>
+  </IllustratedMessage>
+</DropZone>
+```
+
+### Accessibility
+
+A visual label should be provided to `DropZone` using a `Text` element with a `label` slot. If it is not provided, then an `aria-label` or `aria-labelledby` prop must be passed to identify the visually hidden button to assistive technology. 
+
+### Internationalization
+
+In order to internationalize a drop zone, a localized string should be passed to the `aria-label` prop.
```

Comment:
There's a visible label as well which is an option. See https://reactspectrum.blob.core.windows.net/reactspectrum/48e8c567fffac129b7c3f07bdb43ad6b01c68dbe/docs/react-spectrum/Checkbox.html#internationalization